spring-integration-core/src/test/java/org/springframework/integration/dsl/routers/RouterTests.java

```diff
@@ -612,6 +613,21 @@ public void testNestedScatterGatherSuccess() {
 
 	}
 
+	@Timeout(11000)
```

Comment:
This is redundant. Since we are going to fail anyway when ` replyChannel.receive(10000)` returns `null` after those 10 seconds.

Reply:
HI @artembilan,

When Thread is in hung state due to any issue, the test case is going into hung state. This is safety precaution to ensure test case atleast fails instead of making the whole test suite in hung state forever.

Reply:
This comment has not been addressed.

In some future I'm going to remove this `@Timeout` because I will find the code in the test fully sufficient.
Some comment for this option would save it from removal 😉